utils/forest_utils/src/db/file_backed_obj.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::path::PathBuf;
+
+use log::warn;
+
+pub struct FileBacked<T: FileBackedObject> {
+    inner: Option<T>,
```

Comment:
I think we can always provide a sensible default value.
```suggestion
    inner: T,
```

Reply:
Fixed